docs/v1/use-cases.md

```diff
@@ -3,6 +3,13 @@
 * TODO: scrub doc post glossary
 * TODO: add CF SB API as the lingua franca for blackbox services
 
+## Personas
+
+* User
+* Developer
```

Comment:
I suspect that just "Application Manager" (or Operator perhaps) might be sufficient to cover both of these.